test_scripts/Resumption/InteriorVehicleData/021_InteriroVD_resumption_2_apps_same_module_error_with_delay_to_another_rpc.lua

```diff
@@ -40,7 +40,7 @@ local function checkResumptionData()
   EXPECT_HMICALL("UI.AddSubMenu")
   :Do(function(_, data)
       local function sendResponse()
-        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        common.getHMIConnection():SendResponse(data.id, data.method, "GENERIC_ERROR", {})
```

Comment:
must be `common.getHMIConnection():SendError(data.id, data.method, "GENERIC_ERROR", "Error message")`

Reply:
@GetmanetsIrina fixed in a53478ec48ed328414214e025e68d4b3c8ab535d